### PR TITLE
[Docs] Update references from Py.Cafe to PyCafe

### PR DESCRIPTION
**Pull request by LydiaPitts**
## Description
Fix for https://github.com/mckinsey/vizro/issues/750 "Correct all mentions of Py.Cafe"
- Updates reference from Py.Cafe to PyCafe to describe their organization
- Mantians references to https://py.cafe/ to talk about the site itself

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/docs/pages/tutorials/first-dashboard.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ Click on the **Run and edit this code in Py.Cafe** link below to live-edit the d
 <!-- vale off -->
 ## Can I break your code?
 <!-- vale on -->
-When you click the link to "Edit live on Py.Cafe" the dashboard is running inside your browser. Any changes you make are local and you don't need to worry about breaking the code for others. Nobody else sees the changes you make unless you save a copy of the project as your own Vizro py.cafe project.
```
Comment (LydiaPitts) [inline vizro-core/docs/pages/tutorials/first-dashboard.md:40]: Hello! Maybe a silly uncertainty, but I'm not 100% sure if py.cafe is a file naming convention or if it's referring to PyCafe and should be updated. I updated it, but can easily revert

Comment (lingyielia) [reply vizro-core/docs/pages/tutorials/first-dashboard.md:40]: Hi @LydiaPitts I think your judgement is correct. 👍 This is not talking about the site itself.

**Review by lingyielia — APPROVED** [review]
Thanks for contributing! Could you change the PR title to start with `[Docs]`? 🙏

File: vizro-core/docs/pages/tutorials/first-dashboard.md (hunk 1 above)
Comment (LydiaPitts) [reply vizro-core/docs/pages/tutorials/first-dashboard.md:40]: Great, thanks :)

**Comment by LydiaPitts** [comment]
I've updated the title to include docs!

**Comment by stichbury** [comment]
Thank you @LydiaPitts for this awesome contribution! I am reviewing it now and will take it forward to get it merged ASAP.

File: vizro-core/docs/pages/tutorials/first-dashboard.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # A first dashboard
 
-There is no setup needed for your first dashboard, thanks to the amazing [Py.Cafe](https://py.cafe/).
+There is no setup needed for your first dashboard, thanks to the amazing [PyCafe](https://py.cafe/).
 
-Click on the **Run and edit this code in Py.Cafe** link below to live-edit the dashboard.
+Click on the **Run and edit this code in PyCafe** link below to live-edit the dashboard.
```
Comment (stichbury) [inline vizro-core/docs/pages/tutorials/first-dashboard.md:5]: The link created by MkDocs plugin actually says "Py.Cafe" at present but I've asked the creators to accept a PR to change it to "PyCafe" so it is consistent with these changes to the docs.

**Review by stichbury — APPROVED** [review]
Thanks @LydiaPitts for a very thorough and accurate PR. Congratulations on your first contribution to Vizro. 🏆 🥇 🌟

**Pull request merged**